lib/suites/LinkedDataSignature.js

```diff
@@ -50,6 +70,53 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
     this.useNativeCanonize = useNativeCanonize;
   }
 
+  /**
+   * See constructor docstring for param details.
+   *
+   * @returns {{verificationMethod: string, key: LDKeyPair, signer: Function,
+   *   verifier: Function}} - Validated and initialized key-related parameters.
+   */
+  _validateVerificationMethod({verificationMethod, key, signer, verifier}) {
+    if(!(key || signer || verifier)) {
+      // eslint-disable-next-line max-len
```

Comment:
-1 to disabling linting in this use case.

Reply:
K, will fix. (But I would like to have a discussion about it over at https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/50 )

Reply:
Fixed.